### PR TITLE
No longer able to BoH bomb while stunned

### LINKED ISSUE #9173: BoH dedonation while stunned
Aparrently if you try to put one BoH into another and the confirmation appears, even if you are stunned, if you click yes it still makes a singulo.

### PR DESCRIPTION
**Pull request by TheVekter**
Fixes #9173

### CODE REVIEW

**Comment by ike709** [comment]
Fixes feature

**Comment by TheVekter** [comment]
Unless a maintainer says otherwise, it's a bug. You drop the bag when you
get stubbed.

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 1]
```diff
@@ -44,7 +44,7 @@
 /obj/item/weapon/storage/backpack/holding/handle_item_insertion(obj/item/W, prevent_warning = 0, mob/user)
 	if(istype(W, /obj/item/weapon/storage/backpack/holding) && !W.crit_fail)
 		var/safety = alert(user, "You feel this may not be the best idea.", "Put in [name]?", "Proceed", "Abort")
-		if(safety == "Abort" || !in_range(src, user) || !src || !W)
+		if(safety == "Abort" || !in_range(src, user) || !src || !W || user.incapacitated())
```
Comment (Miauw62) [inline code/game/objects/items/weapons/storage/backpack.dm]: replace the last four checks with canusetopic

Comment (TheVekter) [inline code/game/objects/items/weapons/storage/backpack.dm]: I'll give it a shot.

Comment (TheVekter) [inline code/game/objects/items/weapons/storage/backpack.dm]: @Miauw62 That just plain breaks the proc in general. They won't combine anymore. I'll revert it and see if it works normally.

**Pull request merged**